tests/acceptance/tests/Categories/AddLandingPageInCategories.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { test } from '@fixtures/AcceptanceTest';
+import { expect } from '@playwright/test';
+
+test('Shop administrator should be able to create a landing page.', {tag: '@Categories'}, async ({
+    ShopAdmin,
+    IdProvider,
+    TestDataService,
+    AdminCategories, CreateLandingPage, AdminLandingPageDetail, AdminApiContext,
+}) => {
+    const layoutUuid = IdProvider.getIdPair().uuid;
+    const layoutName = `test ${layoutUuid}`;
+    const layoutType = 'landingpage';
+    const landingPageData = {
+        name: `Landing Page ${IdProvider.getIdPair().uuid}`,
+        salesChannel: 'Storefront',
+        seoUrl: `landing-${IdProvider.getIdPair().id}`,
+        status: true,
+    };
+
+    await test.step('Create a landing page layout via API.', async () => {
+
+        await TestDataService.createBasicPageLayout(layoutType, {
+                name: layoutName,
+                id: layoutUuid,
+                type: layoutType,
+            },
+        );
+    });
+
+    await test.step('Create a new landing page and assign layout.', async () => {
+        await ShopAdmin.goesTo(AdminCategories.url());
+        await ShopAdmin.attemptsTo(CreateLandingPage(layoutName, landingPageData, true));
+    });
+
+    await test.step('Verify a new landing page created and assigned layout.', async () => {
+        // Verify created landing page
+        const createdLandingPage = AdminCategories.landingPageItems.locator(`text="${landingPageData.name}"`);
+        await createdLandingPage.click();
+        await AdminLandingPageDetail.loadingSpinner.waitFor({ state: 'hidden' });
+
+        // Verify general tab detail
+        await ShopAdmin.expects(AdminLandingPageDetail.nameInput).toHaveValue(landingPageData.name);
+        await ShopAdmin.expects(AdminLandingPageDetail.landingPageStatus).toBeChecked({ checked: landingPageData.status });
+        await ShopAdmin.expects(AdminLandingPageDetail.salesChannelSelectionList).toHaveText(landingPageData.salesChannel);
+        await ShopAdmin.expects(AdminLandingPageDetail.seoUrlInput).toHaveValue(landingPageData.seoUrl);
+        // Verify layout tab detail
+        await AdminLandingPageDetail.layoutTab.click();
+        await ShopAdmin.expects(AdminLandingPageDetail.layoutAssignmentCardTitle).toHaveText(layoutName);
+        await ShopAdmin.expects(AdminLandingPageDetail.layoutAssignmentCardHeadline).toHaveText(layoutName);
+
+        await ShopAdmin.expects(AdminLandingPageDetail.layoutAssignmentContentSection).toBeVisible();
+        await ShopAdmin.expects(AdminLandingPageDetail.layoutResetButton).toBeVisible();
+        await ShopAdmin.expects(AdminLandingPageDetail.changeLayoutButton).toBeVisible();
+        await ShopAdmin.expects(AdminLandingPageDetail.editInDesignerButton).toBeVisible();
+    });
+
+    await test.step('Clean up the created landing page via API.', async () => {
```

Comment:
That step should the TestDataService do...see here [<code snippet>](https://github.com/shopware/acceptance-test-suite/blob/6b77a5c1c5e3612d2c80c2af55a2fbfb2d730f5b/src/services/TestDataService.ts#L656)

If not we have to adjust the TestDataService method.

Reply:
> ` If not we have to adjust the TestDataService method.`

This test case created 2 data:
1. Cms landing page layout, using TestDataService and is cleaned up also by TestDataService
2. Landing page, using UI steps, which is not cleaned up by TestDataService

This step is to clean up 2. Landing page. 
If I understand you @frobel correctly, we need to use TestDataService to clean up data created not by itself too?

Reply:
you can use that snippet to let the TestDataService delete you landing page.

`TestDataService.addCreatedRecord('landing_page', landingPageId);`


And please check also if your CMS page is also deleted correctly while using that snippet.

Reply:
to extract the URL you can also call: 

`AdminLandingPageDetail.page.url()`

Reply:
`private highPriorityEntities = ['order', 'product', 'landing_page'];` helps me delete landing_page first